### ISSUE TITLE
Invisible divs are created on document body 

### ISSUE THREAD

**Issue by ddorstijn** [isaacHagoel/svelte-dnd-action#259]
![image](https://user-images.githubusercontent.com/1730282/111961459-60294180-8af1-11eb-9212-cf91215d85b0.png)
As seen in the image above. Once I import the dnd library these 3 divs get prepended to the document root. It causes a small flash when I am loading the site. 

I am using the library in a subcomponent. This is visible the moment I import the library.

Is there a reason for this?

**Comment by isaacHagoel** [comment]
This is for accessabily related instructions and alerts. Are you sure this
is what's causing the issue?

**Comment by isaacHagoel** [comment]
@ddorstijn  any updates on this one?

**Comment by ddorstijn** [comment]
Yes, I believe the flash came from a combination of many things. I am using tailwind and async loading. The component flashed for a little bit, but I believe this was due to the placeholder when loading the component. The main error came from tailwinds divider class which a border on elements next to each other. The body was a flex container with main and aside and the invisible elements caused a weird offset in tailwind. I mitigated this by simply not using the divider class but butting a border-left on my aside element.
